src/fmt/str.c

```diff
@@ -140,3 +140,56 @@ size_t str_len(const char *s)
 {
 	return s ? strlen(s) : 0;
 }
+
+
+/**
+ * Convert various possible boolean strings to a bool
+ *
+ * @param val  Pointer to bool for returned value
+ * @param str  String to be converted
+ *
+ * @return int 0 if success, otherwise errorcode
+ */
+int str_bool(bool *val, const char *str)
+{
+	int err = 0;
+
+	if (!val || !str)
+		return EINVAL;
+
+	if (!strcmp(str, "0")) {
+		*val = false;
+	}
+	else if (!strcmp(str, "1")) {
+		*val = true;
+	}
+	else if (!strcmp(str, "false")) {
```

Comment:
Maybe we should ignore case and use `strcasecmp` so variants like `TRUE, FALSE, ...` also works?

Reply:
ok.